src/core/ext/client_config/lb_policy_factory.h

```diff
@@ -47,8 +47,19 @@ struct grpc_lb_policy_factory {
   const grpc_lb_policy_factory_vtable *vtable;
 };
 
+/** A resolved address alongside any LB related information associated with it.
+ * \a user_data, if not NULL, contains opaque data meant to be consumed by the
+ * gRPC LB policy. Note that no all LB policies support \a user_data as input.
+ * Those who don't will simply ignore it and will correspondingly return NULL in
```

Comment:
This seems misleading.  The pick_first and grpclb policies won't just ignore the user data; they will crash if the value is non-NULL.